docs/pages/guides/configuring-statusbar.mdx

```diff
@@ -84,35 +95,39 @@ export default function Playlists() {
+If you use `expo-status-bar` to control your status bar style, the `style="auto"` configuration will automatically pick the appropriate default style depending on the color scheme currently used by the app
+(this is the default behavior, if you leave out the style prop entirely then `auto` will be used). Please note that if you provide a way for users to toggle between color schemes rather than using the operating system theme,
+this will not have the intended behavior, and you should use `style="light"` and `style="dark"` as needed depending on the selected color scheme.
 
 ## Factoring the status bar in with your layout
 
 When you have a translucent status bar, it's important to remember that content can be rendered underneath it (if it couldn't, what would be the point of it being translucent? there would be nothing for you to see through it!).
 
-Libraries like [React Navigation](/guides/routing-and-navigation.mdx) will handle this for you when the UI that they provide overlap with the status bar. You are likely to encounter cases where you will need to manually adjust your layout to prevent some content (such as text) from being rendered underneath it. To do this, we recommend using [react-native-safe-area-context](../versions/latest/sdk/safe-area-context.mdx) to find the safe area insets and add padding or margins to your layout accordingly.
+Libraries like [React Navigation](/guides/routing-and-navigation.mdx) will handle this for you when the UI that they provide overlap with the status bar.
+You are likely to encounter cases where you will need to manually adjust your layout to prevent some content (such as text) from being rendered underneath it.
+To do this, we recommend using [`react-native-safe-area-context`](../versions/latest/sdk/safe-area-context.mdx) to find the safe area insets and add padding or margins to your layout accordingly.
 
 ## Working with misbehaving 3rd-party Libraries
 
-Projects initialized with Expo tools make the status bar `translucent` by default on Android. This is consistent with iOS and more in line with material design. Unfortunately, some libraries don't support `translucent` status bars. This is generally bad practice and those libraries should be fixed, but if you must use one of them, there are some options available for you to accommodate their limitations:
+Projects initialized with Expo tools make the status bar `translucent` by default on Android. This is consistent with iOS and more in line with material design.
+Unfortunately, some libraries don't support `translucent` status bars. This is generally bad practice and those libraries should be fixed,
+but if you must use one of them, there are some options available for you to accommodate their limitations:
 
-### Set the `backgroundColor` of the status bar to an opaque color and disable `translucent` option
+### Workaround #1
```

Comment:
The title here doesn't tell me much when I'm going through it. I'd still like the section headers to be descriptive. I do get the point of having short header titles.

My suggestion here would be to use something like: "Opaque colored background"